src/utils/invariant.ts

```diff
@@ -0,0 +1,15 @@
+const genericMessage = "Invariant Violation";
+
+export class InvariantError extends Error {
+  framesToPop = 1;
+  name = genericMessage;
```

Comment:
These class properties preserve the API of the original https://www.npmjs.com/package/invariant error objects, in case that matters.